src/test/java/nl/tudelft/pixelperfect/gamestates/PlayStateTest.java

```diff
@@ -35,6 +34,7 @@
   private EventScheduler mockScheduler;
   private Spaceship mockSpaceship;
   private DebugHeadsUpDisplay mockHeadsUp;
+  private Settings mockSettings;
```

Comment:
This attribute is called `mockSettings`, whereas you are actually using a real instance of the class instead of a mock object. You should rename it to just settings.